tests/test_api.py

```diff
@@ -338,6 +338,37 @@ def test_properties(self):
         self.assertEqual(self.dbus_props.Get('org.freedesktop.Test.Other', 'color'),
                          'yellow')
 
+        changed_props = []
+        ml = GLib.MainLoop()
+
+        def catch(*args, **kwargs):
+            if kwargs['interface'] == 'org.freedesktop.DBus.Properties' and \
+               kwargs['member'] == 'PropertiesChanged':
```

Comment:
Is there any other signal that would/could happen here? I.e. I wonder if these could  be assertions instead of conditions?

Reply:
There are also `NameOwnerChanged` signals flying by. I changed to assert a bit, but ignore non-Properties signals.